runtime/legion/garbage_collection.h

```diff
@@ -77,7 +77,8 @@ namespace Legion {
       PHYSICAL_MANAGER_REF = 20,
       LOGICAL_VIEW_REF = 21,
       REGION_TREE_REF = 22,
-      LAST_SOURCE_REF = 23,
+      PHYSICAL_TRACE_REF = 23,
```

Comment:
Are we actually planning on making traces distributed collectable objects? If so, what is the use case? If not, there is a simpler garbage collection interface that we can use.